opensearchpy/_async/client/dangling_indices.py

```diff
@@ -25,24 +25,36 @@
 #  under the License.
 
 
+# ----------------------------------------------------
+# THIS CODE IS GENERATED AND MANUAL EDITS WILL BE LOST.
+#
+# To contribute, kindly make essential modifications through either the "opensearch-py client generator":
+# https://github.com/opensearch-project/opensearch-py/blob/main/utils/generate-api.py
+# or the "OpenSearch API specification" available at:
+# https://github.com/opensearch-project/opensearch-api-specification/blob/main/OpenSearch.openapi.json
+# -----------------------------------------------------
+
+
 from .utils import SKIP_IN_PATH, NamespacedClient, _make_path, query_params
 
 
 class DanglingIndicesClient(NamespacedClient):
     @query_params(
-        "accept_data_loss", "master_timeout", "cluster_manager_timeout", "timeout"
+        "accept_data_loss", "cluster_manager_timeout", "master_timeout", "timeout"
     )
     async def delete_dangling_index(self, index_uuid, params=None, headers=None):
         """
-        Deletes the specified dangling index
+        Deletes the specified dangling index.
 
 
-        :arg index_uuid: The UUID of the dangling index
+        :arg index_uuid: The UUID of the dangling index.
         :arg accept_data_loss: Must be set to true in order to delete
-            the dangling index
-        :arg master_timeout (Deprecated: use cluster_manager_timeout): Specify timeout for connection to master
-        :arg cluster_manager_timeout: Specify timeout for connection to cluster_manager
-        :arg timeout: Explicit operation timeout
+            the dangling index.
+        :arg cluster_manager_timeout: Operation timeout for connection
+            to cluster-manager node.
+        :arg master_timeout: Operation timeout for connection to master
```

Comment:
Similar comment for master_timeout comments to mention that it is deprecated. I think it applies to the other generated code PRs.